doc/source/using.rst

```diff
@@ -103,7 +18,16 @@ available as a global variable.
     to handle aspects like ``None`` values and fields that have values
     that may interfere with parsing reStructuredText.
 
+Directives
+==========
+
+.. datatemplate:import-module::
+      :source: sphinxcontrib.datatemplates.domain
+      
+      {% for key,directive in data.DataTemplateDomain.directives.items() %}
```

Comment:
We probably want to sort them so the order is consistent and predictable.

Reply:
Very true, sorting in 73f177e4b085099619e1f503d5fab580db258904